src/tests/async/reflection/reflection-simple.cs

```diff
@@ -2,14 +2,16 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System;
+using System.Linq.Expressions;
 using System.Reflection;
+using System.Reflection.Emit;
 using System.Threading.Tasks;
 using Xunit;
 
 public class Async2Reflection
 {
     [Fact]
-    public static void TestEntryPoint()
+    public static void DynamicInvoke()
```

Comment:
```suggestion
    public static void MethodInfo_Invoke()
```
I would avoid overloading meaning of dynamic, and instead call these test cases according to the API that they exercise.